libr/debug/p/native/linux/linux_coredump.c

```diff
@@ -310,29 +297,24 @@ static bool has_map_anonymous_content(char *buff_smaps, ut64 start_addr, ut64 en
 	return 0;
 }
 
-static bool dump_this_map(char *buff_smaps, ut64 start_addr, ut64 end_addr, bool file_backed, bool anonymous, ut8 perms, ut8 filter_flags) {
-	char *p, *pp, *ppp, *extern_tok, *flags_str;
-	char *identity = NULL;
+static bool dump_this_map(char *buff_smaps, unsigned long start_addr, unsigned long end_addr, 
```

Comment:
function signatures must fit in one line, do not indent like this. if you feel this function is having too many parameters groups them in a struct for info and use only modifiers as arguments, or another struct if there are many arguments.

This is how rust handles default parameters and variable argument functions. and it makes the function signatures more readable.